README.md

```diff
@@ -1 +1,165 @@
-# Uptasticsearch
\ No newline at end of file
+# uptasticsearch
+
+## Introduction
+
+This project tackles the issue of getting data out of Elasticsearch and into a tabular format in R.
+
+# Table of contents
```

Comment:
Should we add a section about supported versions, mention that Auth isn't released yet?